lib/ansible/modules/files/xml.py

```diff
@@ -106,6 +106,12 @@
         the original file back if you somehow clobbered it incorrectly.
     type: bool
     default: 'no'
+  strip_cdata_tags:
+    description:
+      - Remove CDATA tags surrounding text values. This might break your XML
+        file if text values contain characters that could be interpreted as XML
+    type: bool
+    default: 'no'
```

Comment:
add this line:
```
version_added: "2.7"
```
to fix this error
lib/ansible/modules/files/xml.py:0:0: E309 version_added for new option (strip_cdata_tags) should be 2.7. Currently 0.0